pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -179,6 +184,23 @@ func (c *InstallerController) manageInstallationPods(operatorSpec *operatorv1.Op
 		return false, nil
 	}
 
+	// Store success/fail status in the status configmap for revision history tracking
+	statusConfigMap, err := c.kubeClient.CoreV1().ConfigMaps(c.targetNamespace).Get(statusConfigMapName, metav1.GetOptions{})
```

Comment:
still looks like the wrong multiplicity

Reply:
This can probably be reworked, but I don't have the revision number until I start looking at the node statuses, which is where I update the configmap name to be unique for that revision: https://github.com/openshift/library-go/pull/82/files#diff-aea28fda19fe8a3c27c2cf6590e72d89R221

Then once outside the loop I save the configmap